goth/runner/process.py

```diff
@@ -34,20 +35,20 @@ async def run_command(
     if log_prefix is None:
         log_prefix = f"[{args[0]}] "
```

Comment:
I remember from the `yapapi` tests demo that this was logging the full host path to the script, resulting in potentially long prefixes. Perhaps we should truncate the default prefix to the name of the file only?

Reply:
On a second thought: there's already the `log_prefix` parameter that allows the caller to specify a custom prefix if the default one turns out to be too long. Let's not make it more clever than it is now.

Reply:
Moreover, we're going to modify the log-related parts of this function anyway, to enable attaching a log monitor to the command's output (I also have a secret plan to add some :rainbow: colours :rainbow: when logging the output to the console). I'll try to remember about your suggestion when doing those modifications.